modules/admin-ui-frontend/app/scripts/shared/directives/timelineDirective.js

```diff
@@ -1264,6 +1264,122 @@ angular.module('adminNg.directives')
           if (scope.timer) $timeout.cancel( scope.timer );
         });
 
+        /**
+         * Creates a promise object to load the AuthService so that user properties
+         * are available to the current scope.
+         */
+        AuthService.getUser().$promise.then(function (user) {
```

Comment:
I'm not a huge fan of the way this has nested function declarations.  Is this how the rest of Angular codebase is?  It's been a while, but we try to keep things similar, file to file.

Reply:
This is how it was done in a number of different files.